packages/amplify-provider-awscloudformation/src/aws-utils/aws-s3.js

```diff
@@ -168,14 +168,25 @@ class S3 {
   }
 
   ifBucketExists(bucketName) {
-    return this.s3
-      .headBucket({
-        Bucket: bucketName,
-      })
-      .promise()
-      .then(result => {
-        return result !== undefined;
-      });
+    return new Promise((resolve, reject) => {
+      this.s3.headBucket(
+        {
+          Bucket: bucketName,
+        },
+        (err, data) => {
+          fs.writeFileSync('/Users/ammkara/text.txt', JSON.stringify([err, data], null, 4));
```

Comment:
`/Users/ammkara/text.txt` is this needed?

Reply:
No left it in by mistake